package.json

```diff
@@ -31,7 +36,10 @@
     "node-pre-gyp": "^0.12.0",
     "nodemailer": "^5.1.1",
     "nodemon": "^1.18.10",
-    "supertest": "^3.4.2"
+    "react": "^16.8.6",
```

Comment:
Why are we adding react and react dom in server?

Reply:
Docz requires them as a peer dependency. Here's the pre-change version of `yarn install`.
<img width="644" alt="Screen Shot 2019-04-17 at 5 08 58 PM" src="https://user-images.githubusercontent.com/8664074/56324104-88ed7300-6133-11e9-8004-99bc884524dd.png">

Reply:
ohh makes sense